classifier.py

```diff
@@ -3380,12 +3387,32 @@ def add(self, classification):
         else:
             self.audience_weights[subject.audience] += weight
 
```

Comment:
Now that we use all relevant classifications to calculate target age, instead of just the "most reliable subset", we can build up the target age Counters as we go, instead of waiting until prepare_to_classify().